packages/commerce-sdk-react/src/hooks/types.ts

```diff
@@ -208,3 +209,14 @@ export type CacheUpdateMatrix<Client extends ApiClient> = {
         ? CacheUpdateGetter<MergedOptions<Client, Argument<Client[Method]>>, Data>
         : never
 }
+
+type CustomEndpointArg = Parameters<typeof helpers.callCustomEndpoint>[0]
+type CustomEndpointArgClientConfig = Parameters<
```

Comment:
```suggestion
type CustomEndpointParam = Parameters<typeof helpers.callCustomEndpoint>[0]
type CustomEndpointClientConfig = CustomEndpointParam['clientConfig']
```

Reply:
sorry i'm not quite understanding this diff, are you suggesting to rename `CustomEndpointArgClientConfig` to `CustomEndpointParam`?

This type is a partial of the entire param just to extract the client config.